.pre-commit-config.yaml

```diff
@@ -5,6 +5,13 @@ repos:
     hooks:
       - id: check-yaml
         files: .*\.(yaml|yml)$
+  - repo: https://github.com/hadolint/hadolint.git
+    rev: v2.1.0
```

Comment:
```suggestion
    rev: v2.3.0
```

Reply:
I will merge master after your PR merge and fix the version in two places.
Unfortunately, right now we need to specify version in two places.
One - for revision of the hook, second - for the docker hadolint used.